### PR TITLE
docs: added a command to resolve the build error #11549

### LINKED ISSUE #11549: Kubevirt Build Error
**What happened**:
Whenever I am running the `make && make push && make manifests` command it gives the following error
image

**What you expected to happen**:
Proper build of manifest 

**How to reproduce it (as minimally and precisely as possible)**:
Steps to reproduce the behavior.
1. `export DOCKER_PREFIX=index.docker.io/myrepo`
2. `export DOCKER_TAG=mybuild`
3. `make && make push && make manifests`

**Additional context**:

**Environment**:
- KubeVirt version (use `virtctl version`): N/A
- Kubernetes version (use `kubectl version`): 
         Client Version: v1.29.3
         Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
- VM or VMI specifications: N/A
- Cloud provider or hardware configuration: N/A
- OS (e.g. from /etc/os-release): 
          PRETTY_NAME="Ubuntu 22.04.4 LTS"
          NAME="Ubuntu"
          VERSION_ID="22.04"
          VERSION="22.04.4 LTS (Jammy Jellyfish)"
          VERSION_CODENAME=jammy
          ID=ubuntu
- Kernel (e.g. `uname -a`): 
   Linux tushar-HP-Pavilion 6.5.0-26-generic 26~22.04.1-Ubuntu SMP PREEMPT_DYNAMIC Tue Mar 12 10:22:43 UTC 2 x86_64 x86_64 x86_64 GNU/Linux
- Install tools: N/A
- Others: N/A

### PR DESCRIPTION
**Pull request by Tusharjain123**
### What this PR does
It resolves the Build command error while fetching the go container registry during the build process.
Before this PR:

After this PR: 

Fixes #11549 

### Why we need it and why it was done in this way
The following tradeoffs were made: 

The following alternatives were considered:

Links to places where the discussion took place:  https://kubernetes.slack.com/archives/C0163DT0R8X/p1710856660625449

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [x] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note
 
```release-note
"NONE"
```

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ Then build the manifests and images:
 make && make push && make manifests
 ```
 
+**Note:** If the `make && make push && make manifests` command gives an unexpected disconnect or Early EOF while fetching the repositories during the building of manifests and images, then adjusting the default timeout using `export PULLER_TIMEOUT=10000` will solve the issue.
```
Comment (victortoso) [inline docs/getting-started.md:24]: I'd follow the line length of the `Getting Started` section.

I'd rephrase this as something like:

````markdown
**Note:** If you see failures related to fetching some modules, try increasing bazel's timeout with:
 ```bash
 export PULLER_TIMEOUT=10000
 ```
````

**Review by victortoso — COMMENTED** [review]
> Fixes https://github.com/kubevirt/kubevirt/issues/11549

I'm not sure it fixes it, but it is a good workaround.

I'm not sure we can fix it, for instance, by detecting bazel's failure and retry with a longer timeout?

Thanks for the contribution.

File: docs/getting-started.md (hunk 1 above)
Comment (Tusharjain123) [reply docs/getting-started.md]: I have rephrased the line according to your suggestion

**Comment by Tusharjain123** [comment]
> > Fixes #11549
> 
> I'm not sure it fixes it, but it is a good workaround.
> 
> I'm not sure we can fix it, for instance, by detecting bazel's failure and retry with a longer timeout?
> 
> Thanks for the contribution.

Yes, I will try to figure out a permanent solution for this issue

**Comment by xpivarc** [comment]
/approve
@victortoso  ptal
